docs/reference/api/docker_remote_api.md

```diff
@@ -24,6 +24,26 @@ client must have `root` access to interact with the daemon. If a group named
 `docker` exists on your system, `docker` applies ownership of the socket to the
 group.
 
+To interact with the remote API using [cURL](http://curl.haxx.se/download.html) against the unix socket, you must have 
+a version of curl greater than 7.40, after which the --unix-socket flag was added. 
+
+For example, if the Docker daemon is listening on the default socket, then you can
+use curl against the remote API to get information on running containers by:
+```
+curl --unix-socket /var/run/docker.sock http://containers/json
+```
+
+It is also possible to bind the Docker daemon to a TCP port at startup by:
```

Comment:
I wonder if we should mention this at all here, perhaps we should only describe how to connect using the socket (as it's the default).

Also note that we have a section on how to connect using curl in a docker machine setup; https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api.md#using-docker-machine-with-the-api

@moxiegirl wdyt?